internal/packager/extract.go

```diff
@@ -110,12 +109,6 @@ func Extract(name string, ops ...func(*types.App) error) (*types.App, error) {
 	appname := internal.DirNameFromAppName(name)
 	s, err := os.Stat(appname)
 	if err != nil {
-		// URL or docker image
-		u, err := url.Parse(name)
-		if err == nil && (u.Scheme == "http" || u.Scheme == "https") {
-			ops = append(ops, types.WithSource(types.AppSourceURL))
-			return loader.LoadFromURL(name, ops...)
-		}
 		// look for a docker image
 		ops = append(ops, types.WithSource(types.AppSourceImage))
```

Comment:
As the "by-url" is dropped as part of this PR, should'nt we also drop the old image support ? cc @silvin-lubecki

Reply:
Was this resolved by a change elsewhere?